From 1d7da4efefb25130c404fd063a15e54d2c2dd00e Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Fri, 31 Jul 2026 18:09:53 +0200
Subject: [PATCH 15267/15333] arm64: dts: qcom: sdm845-oneplus: Move
 touchscreen y-axis to the right place

This value belongs only to enchilada, fajita sets it's own (as the width
/ y-axis differs).

Signed-off-by: David Heidelberg <david@ixit.cz>
---
 arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi   | 1 -
 arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts | 4 ++++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
index 75ed3974de51..6dc6fa920f5f 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-common.dtsi
@@ -817,7 +817,6 @@ rmi4-f01@1 {
 		rmi4_f12: rmi4-f12@12 {
 			reg = <0x12>;
 			touchscreen-x-mm = <68>;
-			touchscreen-y-mm = <144>;
 			syna,sensor-type = <1>;
 			syna,rezero-wait-ms = <200>;
 		};
diff --git a/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts b/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
index 5f01115e2322..92a4877bc149 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-oneplus-enchilada.dts
@@ -124,6 +124,10 @@ codec {
 	};
 };
 
+&rmi4_f12 {
+	touchscreen-y-mm = <144>;
+};
+
 &wcd9340 {
 	qcom,micbias1-microvolt = <1800000>;
 	qcom,micbias2-microvolt = <2700000>;
-- 
2.55.0

